Vagrantfile

```diff
@@ -48,8 +49,8 @@ Vagrant.configure("2") do |config|
   config.ssh.insert_key = false
 
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 308.0.1"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = $image_version
```

Comment:
this line causes an issue if $image_version is the default value "current", since "current" is not a version-like string. Just wrap this line in a conditional asserting $image_version != "current"